chainerx/_docs/routines.py

```diff
@@ -807,6 +807,13 @@ def _docs_linalg():
 Returns:
     :class:`~chainerx.ndarray`: Output array. Cholesky factor of ``a``.
 
+Note:
+    This functions assumes that ``a`` is symmetric and is undefined otherwise.
```

Comment:
`undefined` sounds `ValueError` from the math perspective or undefined behavior from the C++ perspective. I suggest:

```This functions assumes that ``a`` is symmetric. The output of the forward/backward computation is unspecified otherwise.```

IMO, the **backward of** `eigh` is also returning an inconsistent value unless the forward input is symmetric.

Reply:
Addressed this too.